lighthouse-cli/test/smokehouse/seo/expectations.js

```diff
@@ -174,4 +174,18 @@ module.exports = [
       },
     },
   },
+  {
+    requestedUrl: BASE_URL + 'seo-tap-targets.html',
+    finalUrl: BASE_URL + 'seo-tap-targets.html',
+    audits: {
+      'tap-targets': {
+        score: 0.9, // 10 passing targets/11 total visible targets
+        details: {
+          items: {
+            length: 1,
```

Comment:
Added an assertion for the item details.

It would be nice to assert the full list of detected tap targets from the artifacts. Currently smokehouse just provides the LHR though.